File: NeighborhoodServices/NeighborhoodServices/NeighborhoodServices/DTOs/BookingDto.cs
namespace NeighborhoodServices.DTOs
{
    public class BookingDto
    {
        public int Id { get; set; }
        public string ServiceTitle { get; set; }
        public string ProviderName { get; set; }
        public DateTimeOffset ScheduledTime { get; set; }
        public string Status { get; set; }
        public string CustomerName { get; set; } = string.Empty;
        public string CustomerEmail { get; set; } = string.Empty;
    }

}

File: NeighborhoodServices/neighborhood-ui/src/app/models/booking.dto.ts
export interface BookingDto {
  id: number;
  serviceTitle: string;
  providerName: string;
  scheduledTime: string; // Use `string` if it's ISO format from backend (e.g., "2025-06-22T14:30:00Z")
  status: string;
  customerName: string;
  customerEmail: string;
}
